packages/react-client/src/ReactFlightClientConsoleConfigBrowser.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+const badgeFormat = '%c%s%c ';
+// Same badge styling as DevTools.
+const badgeStyle =
+  // We use a fixed background if light-dark is not supported, otherwise
+  // we use a transparent background.
+  'background: #e6e6e6;' +
+  'background: light-dark(rgba(0,0,0,0.1), rgba(255,255,255,0.25));' +
+  'color: #000000;' +
+  'color: light-dark(#000000, #ffffff);' +
+  'border-radius: 2px';
+const resetStyle = '';
+const pad = ' ';
+
+export function printToConsole(
+  methodName: string,
+  args: Array<any>,
+  badgeName: string,
+): void {
+  let offset = 0;
+  switch (methodName) {
+    case 'dir':
+    case 'dirxml':
+    case 'groupEnd':
+    case 'table': {
+      // These methods cannot be colorized because they don't take a formatting string.
+      // eslint-disable-next-line react-internal/no-production-logging
+      console[methodName].apply(console, args);
+      return;
+    }
+    case 'assert': {
+      // assert takes formatting options as the second argument.
+      offset = 1;
+    }
```

Comment:
Do we allowlist these methods somewhere? I wonder what the behaviour will be if some non-standard console methods are called, like `console.timeStamp` or `console.createTask`?

Reply:
I think I found the answer - https://github.com/facebook/react/pull/28384/files#r1495197550

Reply:
Yes, we only patch a subset. The list and what was excluded is commented on here:

https://github.com/facebook/react/pull/28384#discussion_r1495197550